client/src/menu/game/HeaderMenu.tsx

```diff
@@ -87,14 +90,30 @@ function Information(): ReactElement {
         return myIndex === undefined ? undefined : players[myIndex]?.toString()
     }, [myIndex, players])
 
+
+    const timeLeftText = useMemo(() => {
+        if (timeLeftMs >= 1000000000000000000) {
```

Comment:
cant check if === Infinity?? This seems weird. You can ignore this nitpick if you want to

Reply:
It's not infinity, that's why. The server sends over a u64, which doesn't have Infinity as an option. If you remove this check, you'll see it renders a number around 9 quadrillion

Reply:
ok fine

Reply:
I would rather you make a way to make a phase actually infinite length, so there isnt extra data and stuff. It doesn't really matter too much but it would be far cleaner and future proof.

Reply:
The game stops ticking when recess starts, so it effectively is infinite length